### PR TITLE
Add OKP4 Nemeton testnet

### PR DESCRIPTION
**Pull request by amimart**
Add the OKP4 Nemeton testnet.

### CODE REVIEW

File: testnets/okp4testnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            "denom": "know",
+            "exponent": 6
+          }
+        ],
+        "base": "uknow",
+        "name": "OKP4 Network",
+        "display": "know",
+        "symbol": "KNOW",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/okp4testnet/images/okp4.svg"
```
Comment (JeremyParish69) [inline testnets/okp4testnet/assetlist.json:22]: I won't require it, but any chance you have an image file with square aspect ratio?

File: testnets/okp4testnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,149 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "okp4testnet",
+  "chain_id": "okp4-nemeton-1",
+  "website": "https://okp4.network/",
+  "pretty_name": "OKP4 Nemeton",
+  "status": "live",
+  "network_type": "testnet",
+  "bech32_prefix": "okp4",
```
[the remark this answers is not in the document]
Comment (amimart) [reply testnets/okp4testnet/chain.json:9]: Totally, thanks :)

File: testnets/okp4testnet/assetlist.json (hunk 1 above)
Comment (amimart) [reply testnets/okp4testnet/assetlist.json]: Yes I've updated the logo with a square one.

File: testnets/okp4testnet/chain.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (antho31) [reply testnets/okp4testnet/chain.json:9]: Reserved (https://github.com/satoshilabs/slips/commit/7e6b138e18f1e4b23f25ddf3d1ac87e7b334502a) 
Thanks @JeremyParish69 :)

**Pull request merged**